pkg/asset/agent/manifests/extramanifests.go

```diff
@@ -0,0 +1,60 @@
+package manifests
+
+import (
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/pkg/errors"
+)
+
+const (
+	directory = "openshift"
```

Comment:
This is visible everywhere in the manifests package, so it could use a more specific name.

Reply:
I've reused the same one used elsewhere for consistency, but it seems a valid comment.